examples/design-system-tutorial/src/components/navigation.ts

```diff
@@ -0,0 +1,61 @@
+import {
+    fillColor,
+    neutralLayer2,
+    neutralLayer3,
+    neutralLayer4,
+    neutralStrokeRest,
+    strokeWidth,
+} from "@fluentui/web-components";
```

Comment:
Are our docs the best place for a Fluent example? Does that create clarity or confusion? Perhaps we can enumerate more of the intent in the readme? If this is intended to help those on the Fluent side, I think perhaps we should consider having it exist there.

Reply:
Is it more clear if we just use fast-components? I don't think we should whole-sale just move this over to fluent.

FWIW we use fluent today in the todo example, so this isn't introducing a new pattern.

Reply:
I put the same content above. Now that I've gone through more of the code it looks like "fluent" -> "fast" is the strongest option, and one I suggest we take.